scripts/make-live.R

```diff
@@ -49,9 +49,9 @@ infiles <- c(file.path(root_dir, "intro-to-R-tidyverse",
                          "05-dimension_reduction_scRNA-seq.Rmd")),
              file.path(root_dir,  "machine-learning",
                        c("01-openpbta_heatmap.Rmd",
-                         "02-openpbta_consensus_clustering.Rmd",
-                         "03-openpbta_PLIER.Rmd",
-                         "04-openpbta_plot_LV.Rmd")),
+                         "02-openpbta_consensus_clustering.Rmd")),
+                        #  "03-openpbta_PLIER.Rmd",
+                        #  "04-openpbta_plot_LV.Rmd")),
```

Comment:
File an issue please? 😄  Definitely agree that this is worth looking into, but until we are going to teach this notebook it is firmly under "nice to have" in my mind!

Reply:
Was definitely do that if we are moving forward with this, but I guess it makes sense to file either way!